inc/simcard.class.php

```diff
@@ -317,12 +311,10 @@ function post_addItem() {
    }
 
    function getSearchOptions() {
-      global $CFG_GLPI, $LANG;
-
       $tab = array();
       $tab['common']             = __s('SIM card', 'simcard');
 
-      $tab[1]['table']           = $this->getTable();
+      $tab[1]['table']           = static::getTable();
```

Comment:
I do not agree with this one; see http://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html#static-or-non-static